frontend/packages/ceph-storage-plugin/src/utils/block-pool.tsx

```diff
@@ -60,18 +60,20 @@ export const PROGRESS_STATUS = (t: TFunction): ProgressStatusProps[] => [
   },
   {
     name: POOL_PROGRESS.NOTREADY,
-    icon: LockIcon,
-    desc: t(
-      'ceph-storage-plugin~The creation of an OCS storage cluster is still in progress or has failed. Please try again after the storage cluster is ready to use.',
-    ),
-    className: '',
+    icon: ExclamationCircleIcon,
+    desc: t('ceph-storage-plugin~Pool {name} got created with non ready state'),
+    className: 'ceph-block-pool__error-icon',
   },
   {
     name: POOL_PROGRESS.NOTALLOWED,
     icon: LockIcon,
-    desc: t(
-      "ceph-storage-plugin~Pool management tasks are not supported for default pool and Openshift Container Storage's  external mode.",
-    ),
+    desc: isClusterReady
+      ? t(
+          "ceph-storage-plugin~Pool management tasks are not supported for default pool and Openshift Container Storage's  external mode.",
+        )
+      : t(
+          'ceph-storage-plugin~The creation of an OCS storage cluster is still in progress or has failed. Please try again after the storage cluster is ready to use.',
+        ),
```

Comment:
NOTALLOWED - default pool, external cluster, cluster is unhealthy

Reply:
In UI we try to align the statuses/phases with what we receive from backend, instead of creating new names.
Please try to do that otherwise it creates issues while debugging.

Reply:
Pool state we are aligned with CR

export enum POOL_STATE {
  READY = 'Ready',
  RECONCILE_FAILED = 'ReconcileFailed',
  FAILURE = 'Failure',
}

POOL_PROGRESS is just for changing different body and footer in our pool modal. it is not for CR.

Reply:
Why do we need different statuses for UI ? And if required, please add comments for what each is used for.

Reply:
In a single module, we need to show different body for 
 1. Pool creation form - (Before pool CR, means no pool state)
 2. Block pool creation page for when the cluster is not ready, external cluster, default pool (Before pool CR, means no pool state)
 3. Pool creation failed (Before CR creation and after CR creation)
 4. operation Timed out (Before CR creation)
 5. pool creation in-process (During CR creation )
 6. Pool creation success (After CR creation)
 
 All of these are different bodies on the same module. We can't take decisions using the pool CR only.

Reply:
POOL_PROGRESS has a description on it: https://github.com/openshift/console/pull/8689/files#diff-8fb0182943121a26dbf55cf34d3497e3026c4cdcb01d7d1679dcbf0a5998fc95R34

Reply:
okay, thanks